tests/unit/utils/get-package-json.test.mjs

```diff
@@ -0,0 +1,38 @@
+import { readFile } from 'fs/promises'
+
+import { beforeEach, describe, expect, test, vi } from 'vitest'
+
+import getPackageJson from '../../../src/utils/get-package-json.mjs'
+
+vi.mock('fs/promises', async (importActual) => {
+  const fs = await importActual()
+
+  return {
+    ...fs,
+    readFile: vi.fn(fs.readFile),
+  }
+})
+
+describe('getPackageJson', () => {
+  beforeEach(() => {
+    vi.clearAllMocks()
+  })
+
+  test('should return the package.json of netlify-cli', async () => {
+    const packageJson = await getPackageJson()
+
+    expect(packageJson).not.toBeUndefined()
+    expect(packageJson.name).toBe('netlify-cli')
+
+    expect(readFile).toHaveBeenCalledOnce()
+  })
+
+  test('should not reread package.json', async () => {
```

Comment:
nit: It might be nice to somehow indicate that this test is coupled to the previous one, as the only reason it doesn't re-read `package.json` is that the previous test triggers the first read.